ui/layouts/MobileApp.tsx

```diff
@@ -45,11 +45,13 @@ const Limit = styled('div')(
     display: 'flex',
     flexDirection: 'column',
     flex: '1 0 auto',
-    p: 8,
+    px: 8,
+    py: 12,
```

Comment:
This component is used globally. @bitboxer what do you think?

Reply:
Yeah, right. Will revert that change.